src/tests/multihost/alltests/test_sssctl_local.py

```diff
@@ -52,12 +61,18 @@ def test_0002_bz1599207(self, multihost, backupsssdconf, localusers):
         multihost.client[0].service_sssd('stop')
         tools.remove_sss_cache('/var/lib/sss/db')
         tools.remove_sss_cache('/var/log/sssd')
-        rm_cmd = 'rm -f %s' % SSSD_DEFAULT_CONF
-        multihost.client[0].run_command(rm_cmd, raiseonerr=False)
+        if not client_version(multihost):
```

Comment:
This is a sssctl test to check local users. I think it should run for rhel9 by enabling file domains else run for rhel8 by deleting conf file.

Reply:
Changes done